docs/it-manual/monitoring.md

```diff
@@ -0,0 +1,719 @@
+# Monitoring a CiviForm deployment
+
+CiviForm supports server monitoring via [Prometheus](https://prometheus.io/) metrics visualized in [Grafana](https://grafana.com/).
+
+![](<../.gitbook/assets/monitoring-architecture.png.png>)
+
+## Enabling metrics export
+
+Exporting metrics from the server is optional, and must be enabled by setting the `CIVIFORM_SERVER_METRICS_ENABLED` environment variable to `true`. When enabled, the server exports metrics from the `/metrics` HTTP route. While these metrics data are not sensitive, it is a good practice to prevent access to this route from the public internet (which is done by default when using the [CiviForm terraform deployment system](https://github.com/civiform/cloud-deploy-infra/)).
+
+## Viewing metrics
+
+### AWS
+
+The [CiviForm terraform deployment system](https://github.com/civiform/cloud-deploy-infra/) for AWS deploys the monitoring stack automatically. After deployment, user access to the Grafana dashboard and configuration of the dashboard need to be done manually.
+
+#### Configure access
+
+[AWS Managed Grafana](https://aws.amazon.com/grafana/) uses [AWS IAM Identity Center](https://aws.amazon.com/iam/identity-center/) for access management. **Note that this is a different service from [AWS IAM](https://aws.amazon.com/iam/) the accounts/user profiles in AWS IAM Identity Center are completely separate from accounts in AWS IAM.**
+
+1. Login to the AWS console and navigate to the IAM Identity Center service
+1. In the left nav, click "Users"
+1. For each user you'd like to grant access to viewing metrics, click the "Add user" button and follow the workflow
+1. Follow instructions [here](https://docs.aws.amazon.com/grafana/latest/userguide/AMG-manage-users-and-groups-AMG.html) for adding granting access for those users to your Grafana workspace
```

Comment:
I think it is worth calling out here that after adding a user, you'll need to separately make them an admin in the grafana workspace assigned users configuration

Reply:
Done